README.md

```diff
@@ -168,8 +168,8 @@ return [
     ],
     'validators' => [
         'factories' => [
-            Validator\DbRecordExists::class => Validator\Db\RecordExistsFactory::class,
-            Validator\DbNoRecordExists::class => Validator\Db\NoRecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbRecordExists' => Validator\Db\RecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbNoRecordExists' => Validator\Db\NoRecordExistsFactory::class,
```

Comment:
@akrabat More concisely, version 1.0.0 shipped with the name ZF\ContentValidation\Validator\DbRecordExists as the factory mapping in the config. (https://github.com/zfcampus/zf-content-validation/blob/1.0.0/config/module.config.php#L31)

Whether this was intentional or an oversight initially, we now need to maintain this within the 1.x releases to keep BC.